### ISSUE TITLE
Builders Wand Block duplication glitch still existing

### ISSUE THREAD

**Issue by modsareawesome** [rwtema/extrautilities_old_issues#1700]
hi there, i am using Extra Utilities 2 on 1.10.2 without any other mods affecting the mod or its items (like better Builders Wands etc.)

I am using the newest file of the mod. (1.6.3) 
[https://minecraft.curseforge.com/projects/extra-utilities/files/2481037]

But i have still experienced of the Bug with the builders wand, where i placed some cobblestone blocks for a dark room  spawner and noticed while building with the wand that no ressources were token.

I already checked the config, but it seems to be ok.

I still were searching on the web and found a closed issue that was handling the same problem too. (thats why i updated the mod)
[https://github.com/scalda/FoolCraft/issues/404]

But the bug still exists, and it ruins the gameplay, because it is cheating for me and i still stopped using the BW. But i planned some projects where i need it! Please fix it.

Greetings, xXL31FXx

**Comment by Twisted-Code** [comment]
duplicate of probably about 20 other open issues that probably will never get fixed unless a certain developer starts going through them. It's annoying @rwtema , I know, but things like this are simple fixes that would go a long way. This one pretty much forces server admins to either ban the BW or allow people to duplicate diamonds and any other material that has a block form. 

Seriously, all you have to do is add a tiny bit of code to the item's onRightClick method (whatever it is actually named) to reduce whatever stack you are already detecting before you render the yellow overlay by however many blocks are rendered in that overlay. it's not rocket science, and it's barely computer science. I'd do it myself as a pull request but the proprietary/closed source nature of the mod prevents it.

**Comment by Twisted-Code** [comment]
in the meantime (or forever if it takes that long), I'm going to disabled the item and try replacing it with Better Builder's Wands (by Portablejim). Hopefully RWTema can find time to fix this. I'm not sure why it's even still an issue (given the apparent simplicity and the unfortunate impact of dupe bugs), but thankfully there are alternatives.

**Comment by Twisted-Code** [comment]
(and again, I want to at least try to fix it myself, but I need to see the code in question before I can, and that's not really possible with closed source projects unless RWT is willing to make an exception here)

**Comment by tenten8401** [comment]
I can confirm this is an issue as well.

**Comment by hron84** [comment]
@macks2008 I recommend you to try reaching out Tema in Twitter, he more likely reads that platform. Still no guaranties, but you know... better than nothing.

**Comment by modsareawesome** [comment]
Okay, i still installed better builders wands and it works fine. Thank You for your help

**Comment by tenten8401** [comment]
@modsareawesome This is still a problem with this mod specifically, that doesn't mean just close it because you found a replacement.

**Comment by hron84** [comment]
@modsareawesome yeah, please reopen this issue as it is still exists and replacing a single item with an item from another mod is not neccessarily solves _this_ problem.

**Comment by WhatThePuckPT** [comment]
I may have figured out the issue - the Builders Wand functions are swapped between **Survival** and **Creative**. I noticed after some testing that it uses up the blocks in Creative, then swapped to Survival and no blocks were used. Using Extra Utilities 2 1.6.3.

**Comment by tenten8401** [comment]
I was actually just messing with this the other day and saw the same behavior.
